tools/install_demo_configuration.sh

```diff
@@ -186,7 +186,7 @@ echo "Elasticsearch lib dir: $ES_LIB_PATH"
 echo "Detected Elasticsearch Version: $ES_VERSION"
 echo "Detected Open Distro Security Version: $SECURITY_VERSION"
 
-if $SUDO_CMD grep --quiet -i opendistro_security "$ES_CONF_FILE"; then
+if $SUDO_CMD grep --quiet -i opendistro-security "$ES_CONF_FILE"; then
```

Comment:
opendistro security settings are not renamed

Reply:
Yes, we're not renaming the configurations. Chaned it back.